src/v3/src/util/idxUtils.ts

```diff
@@ -62,6 +63,26 @@ export const getUserInfo = (transaction: IdxTransaction): UserInfo => {
   return user.value as UserInfo;
 };
 
+export const getUserProvidedUserInfo = (data: FormBag['data']): UserInfo => {
```

Comment:
Is it possible to re-use the `getUserInfo` function? Or is it very different to get data from the Transaction vs the FormBag?

Reply:
Yes it is inherently different, the transaction stores the userInfo exactly as the interface dispays (`{ identifier: ..., profile: { firstName: ..., lastName: ... }}`. Whereas the user related data the is provided when the user is in enroll-profile flow is not available in the transaction but instead set in the formBag by form entry and the keys are `userProfile.email`, `userProfile.firstName`, `userProfile.lastName`, etc. I only created this convenience function `getUserProvidedUserInfo` to avoid having to modify the the functions the rely on userInfo.